minotaur.md

```diff
@@ -1,3 +1,84 @@
+
+## Contributing a package recipe
+
+### 1. Fork the minotaur-recipes repository, and clone the fork it locally
+
+To contribute a package recipe, you will need to fork the [minotaur-recipes](https://github.com/poseidon-framework/minotaur-recipes/tree/main) repository, and clone it somewhere on your local machine, so you can add files to it. 
+You can follow the steps on [this guide](https://docs.github.com/en/get-started/quickstart/fork-a-repo), but make sure you use the SSH key option when cloning the repository locally.
+
+### 2. Create a folder for the recipe
+
+You will now need to create the directory for the package recipe you want to provide. The name of the directory should match the name of the package, and follow the poseidon package naming scheme that includes the year of the publication, the last name of the first author, as well as an identifier for the package all separated by underscores (e.g. `2018_Lamnidis_Fennoscandia`).
+
+```bash
+## Run this within your local clone of the minotaur-recipes repository
+mkdir packages/<PACKAGE_NAME> ## Where <package_name> is replaced by the name of the package recipe you are adding.
+```
+
+### 3. Create the SSF
+
+Now that you have the directory for the package recipe, and the name of the resulting package, you will need to create the Sequencing Source File (SSF) for the package. We offer a convenient script for this, which creates an SSF file including all columns required by Minotaur from an ENA project accession number. You can run the script like this:
+
+```bash
+scripts/create_ssf_from_ena_project.py -o packages/<PACKAGE_NAME>/<PACKAGE_NAME>.ssf <ENA_PROJECT_ACCESSION>
+```
+
+Where `<ENA_PROJECT_ACCESSION>` is the accession number of the ENA project you want to create the SSF for, and `<PACKAGE_NAME>` is the name of the package recipe you are adding.
+This will create the file `<PACKAGE_NAME>.ssf` in the directory you created in the previous step.
+
+### 4. Fill-in the SSF
+
+While most fields of the SSF file have been filled-in automatically with information from the ENA, the `udg`, `library_built`, and `poseidon_IDs` columns need to be specified manually. 
+  - The `udg` column should be filled with the UDG treatment of the samples (`plus`, `half` or `minus`)
+  - The `library_built` column should be filled with the strandedness of each library (`ds` or `ss`)
+  - The `poseidon_IDs` column should be filled with the Poseidon IDs that this set of data should be associated with. If one set of reads should be associated with multiple Poseidon IDs, these should be separated by a semi-colon (e.g. `GUP001;GUP001_SG`)
+  - The optional `notes` field, where you can record any notes you might have about the data, or the decisions you made if any (e.g. `Data in the FastQ is actually a mix of UDG-half and non-UDG libraries. Setting to udg to "none".`)
```

Comment:
This long inline code renders weirdly on the website. Maybe a shorter example is sufficient or we wrap it in `*`, not back-ticks.

Reply:
You can run `docsify serve` to preview the website.